src/main/webapp/app/exercises/programming/manage/update/programming-exercise-update.component.html

```diff
@@ -72,6 +72,7 @@ <h2 *ngIf="isImportFromExistingExercise || isImportFromFile" id="jhi-programming
             ></jhi-programming-exercise-grading>
 
             <jhi-exercise-update-notification
+                *ngIf="!isExamMode"
```

Comment:
why?
This is especially relevant for the exam mode

Reply:
The client filters out the notifications containing this text and I did not find them to be used at any other place (especially during the exam).